### ISSUE TITLE
boost.context broken in 1.63 for mingw-w64

### ISSUE THREAD

**Issue by pitti98** [Alexpux/MINGW-packages#2397]
This is a known bug and has already been fixed for the next release:
https://svn.boost.org/trac/boost/ticket/12719

Any chance that the patch is applied and the boost package updated before the next boost release? Or will we have to wait? I don't mean to be demanding, I just want to know.

If it will be included in the next release, I would be grateful for any pointers on how to apply this patch and build the boost package from source myself (rather than just downloading boost and build it regularly as a non-MSYS2-package).

Thank you!
Peter

**Comment by pitti98** [comment]
It seems I'm lucky. I just found out that the next boost release is just around the corner and will be released next wednesday the 19th. I'm assuming the msys2 update will follow shortly after.

Nevertheless, it would be nice to have a howto that explains how to build msys2 packages from source :)

Thanks, and sorry.
Peter

**Comment by mati865** [comment]
Here is explained how to build packages using PKGBUILD: https://github.com/msys2/msys2/wiki/Contributing-to-MSYS2#building-packages

**Comment by pitti98** [comment]
Ok, I have to apologize some more for not googling enough.
I was about to write that I just found it, when your comment popped up while I was typing :)

I checked out the package sources from github:
git clone https://github.com/Alexpux/MINGW-packages.git
and I'm currently building boost. Let's see how that goes, and how easy it is to integrate the patch.

Anyway, it seems my immediate problem is solved, and it will be solved for others in a couple of days with the next release.

Thanks,
Peter
